css/main.css

```diff
@@ -153,7 +174,7 @@ main {
 }
 
 #projects h2 {
-  font-family: "Crete Round", serif;
+  font-family: 'Crete Round', serif;
   font-style: normal;
   font-weight: normal;
```

Comment:
Don't you think we can avoid repeating these properties by putting them in a class and reusing it? 🤔 or we could try adding multiple classes for styling. 👍 
 Example:
```
 .intro-text, #projects h2, li {
  font-style: normal;
  font-weight: normal;
}
```
Please check out this [StackOverflow link](https://stackoverflow.com/questions/7261823/how-to-reuse-styles/7261864) for more insight 🙂